Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-dashboard/rename-category-dialog.html

```diff
@@ -0,0 +1,18 @@
+﻿<div class="dialog">
+    <div class="dialog-title clearfix">
+        <h1 class="pull-left" md-dialog-title>Update Section</h1>
+        <button type="button" class="pull-right" md-dialog-close><i class="material-icons">close</i></button>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)